client/ide/lib/firebaserealtimemanager.coffee

```diff
@@ -4,33 +4,40 @@ IDEMetrics       = require './idemetrics'
 generatePassword = require 'app/util/generatePassword'
 GOOGLE_OATH_SCOPES = 'email, https://www.googleapis.com/auth/drive.metadata.readonly'
 RealtimeManagerFactory = require './realtimemanagerfactory'
+firebase = require("firebase");
 
 module.exports = class FirebaseRealtimeManager extends RealtimeManagerFactory
 
-  constructor: (@KodingApp) ->
+  constructor: (options = {}, data) ->
     super options, data
-    @firebase = new Firebase("https://KodingApp.firebaseio.com")
+    
     @hash = generatePassword 64, no
 
     @collaborativeInstances = []
     @collaborativeEventListeners = {}
     
-    
+    config = {
+      apiKey: "AIzaSyBTRPPijf7UzGW3LUom46ykVhECpmZNRvo",
+      authDomain: "koding-cff25.firebaseapp.com",
+      databaseURL: "https://koding-cff25.firebaseio.com/",
+      storageBucket: "gs://koding-cff25.appspot.com"
+    };
+    firebase.initializeApp(config)
```

Comment:
Config object should be designated in client runtime options. Check out default configuration modules.

[main.default.coffee](https://github.com/koding/koding/blob/master/config/main.default.coffee)
[credentials.default.coffee](https://github.com/koding/koding/blob/master/config/credentials.default.coffee)

[config/generateRuntimeConfig](https://github.com/koding/koding/blob/master/config/generateRuntimeConfig.coffee) module generates configuration object for client application.